pages/en/1_getting-started/6_scenes.md

```diff
@@ -7,23 +7,23 @@ We're going to rebuild our movie set from before but using a scene instead of wr
 <div class="action">
   <p>Create a new Scene asset named `Main Scene` and open it up.
 </div>
-
+<!-- Ok, this opens the "Scene Editor, right? That's what we see when we open this new scene? -->
 <div class="note">
   <p>**In the scene editor, use `WASD`** to move forward, backward, left and right, `Space` to go up and `Shift` to go down.
 </div>
-
+<!-- Okay, move *what*, exactly? The cursor? The crosshairs? WASD does *nothing* when I'm looking at this screen. The user needs more introduction as to what the Scene Editor is, what is does, and how it works. The speaker seems to assume the reader already knows all of these things. -->

```

Comment:
To move your own viewpoint. But since the scene editor is empty by default, you'll see nothing until you enable the grid or create an actor. This should change in the future (we'll enable the grid by default I think)